CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.0.0] - 2020-11-23
+
+### Added
+
+- TypeScript typings ([#4](https://github.com/MetaMask/extension-port-stream/pull/4))
+
+### Removed
+
+- **(BREAKING)** Remove `readable-stream` dependency ([#4](https://github.com/MetaMask/extension-port-stream/pull/4))
+  - Consumers using this package in browser environments will have to bring their own Node.js stream polyfill.

```

Comment:
Here, I updated the "Changed" header to a "Removed" header. Keep a Changelog has different headers for such purposes, and we should try to use them instead of putting everything under "Changed" (like I have mostly done so far 😅).

Also, I added some instructions for how to handle the breaking change, and turned the `#4` into a link, which GitHub does not do automatically in changelogs.